### PR TITLE
Split ARMHF and ARM64 YAML files out to match x86_64

### PR DESCRIPTION
**Pull request by rdimitrov**
Although the YAML files are split, I understand that merging this will be stalled until the blocking issue for building the images is resolved. 

Note: 
I've updated the image versions(not available yet) in the YAML files as well. 
Comment if needed to split these changes in separate PRs, so this one can be taken care of.

## Description

Related to issue - https://github.com/openfaas/faas-netes/issues/361
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

The PR is tested as follows. For each ARM device:
* Installed kubeadm
* Created a single node k8s cluster with kubeadm (w/weave)
* Applied the namespace file
* Applied the yaml files - respectively yaml_armhf or yaml_arm64
* Made sure everything is replicated
* Verified that the OpenFaaS dashboard is accessible
* Deployed the node info function
* Made sure that the metrics are updated as expected
* Deleted the node info function
* Deleted the whole OpenFaaS stack

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: yaml/gateway-dep.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
       serviceAccountName: faas-controller
       containers:
       - name: gateway
-        image: openfaas/gateway:0.9.14
+        image: openfaas/gateway:0.10.2
```
Comment (alexellis) [inline yaml/gateway-dep.yml]: Let's only change one thing at a time please? We are either updating the versions or we're splitting out the YAML files.

Comment (rdimitrov) [reply yaml/gateway-dep.yml]: Resolved

**Comment by rdimitrov** [comment]
Verified the PR on both armhf and arm64 environments 👍

**Comment by alexellis** [comment]
> How Has This Been Tested?
> Not tested yet.

Re: your comment on verification. Can you update the main description please and give details of how you verified?

Note: I'll merge your PR after that, without testing it myself.

**Comment by alexellis** [comment]
Thank you

**Pull request merged**